src/app/pages/api/openai.ts

```diff
@@ -35,7 +49,7 @@ const fetchData = async (): Promise<OpenaiApiType> => {
   return data as OpenaiApiType;
 };
 
-fetchData().then(
+getChatResponse({ knowledge: [] }).then(
   (response) => console.log(response)
   // console.log(response.choices[0].message.content)
```

Comment:
Could we remove this comment?

Reply:
can do for now yes! we could just return the message now but imo easier for debugging if we just return the object, not like it's needed atm anyway